index.php

```diff
@@ -28,6 +29,10 @@
                 <h1>SOLID</h1>
                 <h2>Properties</h2>
             </div>
+            <div class="smallTitle">
+                <h1>SOLID</h1>
+                <h2>Properties</h2>
+            </div>
```

Comment:
Are lines 28-31 and lines 32-35 both meant to be in there? If not, remove one; if so, don't.